### ISSUE TITLE
Auto renew in submission page

### ISSUE THREAD

**Issue by leo900807** [TIOJ-INFOR-Online-Judge/tioj#8]
Auto renew verdict when a submission is created

**Comment by edisonhello** [comment]
Do you mean auto refresh in per-submission’s page until the verdict of that submission is finalized?

**Comment by leo900807** [comment]
Yes, and refresh interval can be longer if the verdict is not validating but queued. It can prevent server overload.
I have tried to use ajax to refresh, but it causes collapse not working

**Comment by adrien1018** [comment]
Added on 923a91ddc7e5606d3d612021994fd1d1869b9725 using ActionCable.
